src/dev-app/index.html

```diff
@@ -23,13 +23,12 @@
   <script src="https://www.youtube.com/iframe_api"></script>
   <script src="https://unpkg.com/@googlemaps/markerclustererplus/dist/index.min.js"></script>
   <script>
-    (function loadGoogleMaps(variables) {
-      var key = variables ? variables.GOOGLE_MAPS_KEY : null;
-      var script = document.createElement('script');
-      script.src = 'https://maps.googleapis.com/maps/api/js?libraries=visualization' +
-        (key ? '&key=' + key : '');
-      document.body.appendChild(script);
-    })(window.DEV_APP_VARIABLES);
+    // Key can be set through the `GOOGLE_MAPS_KEY` environment variable.
+    const key = window.GOOGLE_MAPS_KEY;
```

Comment:
I think that we still need the IIFE here so the constants aren't declared globally.

Reply:
re-added. didn't cause any issues, but still good to encapsulate 👍